src/mod/applications/mod_http_cache/aws.h

```diff
@@ -33,13 +34,27 @@
 #include <switch_curl.h>
 #include "common.h"
 
-/* (SHA1_LENGTH * 1.37 base64 bytes per byte * 3 url-encoded bytes per byte) */
-#define S3_SIGNATURE_LENGTH_MAX 83
+#define DATE_STAMP_LENGTH 9			// 20190729
+#define TIME_STAMP_LENGTH 17		// 20190729T083832Z
+#define DEFAULT_BASE_DOMAIN "s3-%s.amazonaws.com"
```

Comment:
We had to change the base domain format to make this work. New format uses a . rather than a -, so: "s3.%s.amazonaws.com"

Reply:
Updated by commit fd9dfcd92c61b411d4f5720d2f5d7b427cd425ff